<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenu</class>
 <widget class="QDialog" name="SettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Client to Central Server</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Local Server</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Client to Local Server</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>89</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="clientToLocalServerPage">
    <widget class="QPlainTextEdit" name="localServerClientPortText">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>443</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="localServerAddressText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>https://api.citrahold.com</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verifyLocalServerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autodetectLocalServerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Autodetect server on network</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You are connecting to a central server.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="clientToCentralServerPage">
    <widget class="QPlainTextEdit" name="centralServerAddressText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="centralServerPortText">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>29170</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verifyCentralServerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetValuesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset To Default Values</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You are connecting to a server on your network.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="localServerPage">
    <widget class="QPlainTextEdit" name="localServerPortText">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="localServerEnableDisableButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Server is OFF, click to ENABLE.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testLocalServerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QLabel" name="privateIPLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>user's private IP goes here</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="promptDownloadCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prompt before download</string>
     </property>
    </widget>
    <widget class="QPushButton" name="localServerResetPortButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>You are hosting a local server on your network.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>361</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
